Task/python/Taskfile.yaml

```
@@ -0,0 +1,53 @@
+# https://taskfile.dev
+
+version: '3'
+
+vars:
+  IMAGE_NAME: null
+  VERSION: null
+  PYTHON_VERSION: null
+
+tasks:
+  lint:
+    preconditions:
+      - sh: test `git ls-files -z -o --exclude-standard | wc -w` = "0"
+        msg: "There are untracked files, please commit before linting."
+    cmds:
+      - docker run --rm -v ${PWD}:/goat -e RUN_LOCAL=true -e EXCLUDE="/goat" seiso/goat:latest
+
+  test:
+    preconditions:
+      - sh: test `git symbolic-ref -q HEAD`
+        msg: "In detached HEAD state."
+    cmds:
+      - pipenv run pytest tests 2>&1 >/dev/null
```

Shouldn't we allow the test output to be shown in the logs for troubleshooting?
```suggestion
      - pipenv run pytest tests
```